src/pytorch_lightning/callbacks/batch_size_finder.py

```diff
@@ -31,6 +31,8 @@
 class BatchSizeFinder(Callback):
     SUPPORTED_MODES = ("power", "binsearch")
 
+    optimal_batch_size: Optional[int]
```

Comment:
Why is this annotated in the class definition instead of in the `__init__`?

Reply:
@carmocca 
This is my mistake. And my modification wasn't in time.
In the head of my fork branch, this annotation is moved into __init__.
May I create a new PR to move this annotation into __init__ ?

https://github.com/ar90n/pytorch-lightning/commit/7076c1bfc942710d48793a866e3622d8c43f5632

Reply:
Yes, please. Thank you :purple_heart: